fusor-ember-cli/app/routes/review/installation.js

```diff
@@ -17,9 +17,15 @@ export default Ember.Route.extend({
         });
     }
     if (model.get('deploy_openstack')) {
-        controller.set('openstackPlan', this.store.find('deployment-plan', 'overcloud'));
-        controller.set('openstackNodes', this.store.find('node'));
-        controller.set('openstackProfiles', this.store.find('flavor'));
+        this.store.find('deployment-plan', model.get('id')).then(function (plan) {
+            controller.set('openstackPlan', plan);
```

Comment:
it shouldn't be necessary to do `.then()` but for some reason, the promise wasn't resolving.